thunder/core/prims.py

```diff
@@ -4030,6 +4030,8 @@ def embedding_backward_meta(grad, indices, num_weights, padding_idx, scale_grad_
 def copy__meta(
     copy_from: TensorProxy,
     copy_to: TensorProxy,
+    *,
+    grad_enabled: bool = False,
```

Comment:
To be as simple as possible prims should not have default values.

The current choice works well with all existing code but the introduced behavior of raising warnings for nvFuser-executed copies is not working:
 ```py
import torch
import thunder

test = torch.nn.BatchNorm3d(2, track_running_stats=True).to("cuda")
x = torch.ones((1, 2, 3, 4, 12), device="cuda")
jtest = thunder.jit(test)

out = jtest(x) # No warning
print(thunder.last_traces(jtest)[-1]) # We can see in the trace that grad_enabled is set to False
```

Reply:
It would be better not to use a default value here and replace all occurrences of `prims.copy_` with `thunder.torch.copy_`.

Reply:
Removed.